src/pages/Bridge/useBridgeForm.ts

```diff
@@ -0,0 +1,226 @@
+  );
+
+  const validateAmountMantissa = useCallback(
+    (v: string, ctx: z.RefinementCtx) => {
+      const xvsAmountTokens = new BigNumber(v);
+      const xvsAmountUsd = xvsAmountTokens.times(xvsPriceUsd);
+      const isSingleTransactionLimitExceeded = xvsAmountUsd.gt(maxSingleTransactionLimitUsd);
+      const maxSingleTransactionLimitTokens = maxSingleTransactionLimitUsd.dividedBy(xvsPriceUsd);
+      const doesNotHaveEnoughXvs = walletBalanceTokens.lt(xvsAmountTokens);
+
+      if (isSingleTransactionLimitExceeded) {
+        const readableAmountTokens = formatTokensToReadableValue({
+          value: maxSingleTransactionLimitTokens,
+          token: xvs,
+        });
+        const readableAmountUsd = formatCentsToReadableValue({
+          value: convertDollarsToCents(maxSingleTransactionLimitUsd),
+        });
+        ctx.addIssue({
+          code: 'custom',
+          message: t('bridgePage.errors.singleTransactionLimitExceeded.message', {
+            readableAmountTokens,
+            readableAmountUsd,
+          }),
+          path: ['singleTransactionLimitExceeded'],
+        });
+      }
+
+      const isDailyTransactionLimitExceeded = xvsAmountUsd.gt(remainingXvsDailyLimitUsd);
+      if (isDailyTransactionLimitExceeded) {
+        const readableAmountTokens = formatTokensToReadableValue({
+          value: remainingXvsDailyLimitTokens,
+          token: xvs,
+        });
+        const readableAmountUsd = formatCentsToReadableValue({
+          value: convertDollarsToCents(remainingXvsDailyLimitUsd),
+        });
+        const date = fromUnixTime(dailyLimitResetTimestamp.toNumber());
+        date.setDate(date.getDate() + 1);
+        ctx.addIssue({
+          code: 'custom',
+          message: t('bridgePage.errors.dailyTransactionLimitExceeded.message', {
+            readableAmountTokens,
+            readableAmountUsd,
+            date,
+          }),
+          path: ['dailyTransactionLimitExceeded'],
+        });
+      }
+
+      if (doesNotHaveEnoughXvs) {
+        ctx.addIssue({
+          code: 'custom',
+          message: t('bridgePage.errors.doesNotHaveEnoughXvs', {
+            tokenSymbol: xvs?.symbol,
+          }),
+          path: ['doesNotHaveEnoughXvs'],
+        });
+      }
+    },
+    [
+      dailyLimitResetTimestamp,
+      maxSingleTransactionLimitUsd,
+      t,
+      remainingXvsDailyLimitTokens,
+      remainingXvsDailyLimitUsd,
+      walletBalanceTokens,
+      xvsPriceUsd,
+      xvs,
+    ],
+  );
+
+  const formSchema = z.object({
+    fromChainId: z.nativeEnum(ChainId),
+    toChainId: z.nativeEnum(ChainId),
+    amountTokens: z.string().min(1).superRefine(validateAmountMantissa),
+    bridgeEstimatedFeeMantissa: z.custom<BigNumber | undefined>(validateBridgeFeeMantissa),
+  });
+
+  const defaultValues = useMemo(
+    () => ({
+      fromChainId: chainId,
+      toChainId: chains.find(chain => chain.id !== chainId)?.id as ChainId,
+      amountTokens: undefined,
```

Comment:
```suggestion
      amountTokens: '',
```

You need to set this valut to an empty string, otherwise you'll get the following error.
<img width="437" alt="Screen Shot 2024-01-18 at 12 34 24" src="https://github.com/VenusProtocol/venus-protocol-interface/assets/44675210/5d2cc359-f58c-4adf-98a1-3a576eb9ac99">